.swiftlint.yml

```diff
@@ -2,7 +2,36 @@ disabled_rules:
  - type_name
 
 opt_in_rules:
+ - array_init
+ - attributes
+ - closure_end_indentation
+ - closure_spacing
+ - contains_over_first_not_nil
+ - discouraged_optional_boolean
+ - empty_count
+ - fatal_error_message
+ - first_where
  - force_unwrapping
+ - implicit_return
+ - implicitly_unwrapped_optional
+ - joined_default_parameter
+ - literal_expression_end_indentation
+ - multiline_arguments
+ - multiline_parameters
+ - operator_usage_whitespace
+ - overridden_super_call
+ - override_in_extension
+ - pattern_matching_keywords
+ - private_action
+ - private_outlet
+ - prohibited_super_call
+ - redundant_nil_coalescing
+ - sorted_first_last
+ - sorted_imports
+ - trailing_closure
+ - unneeded_parentheses_in_closure_argument
+ - vertical_parameter_alignment_on_call
+ - yoda_condition
```

Comment:
Given that in Swift, contrary to most C-based languages, the affectation of a variable doesn't return the affected value but `Void`, it's not possible to write `if x = 42` instead of `if x == 42` because `x = 42` is an expression returning `Void`. That's one great thing about Swift.

So that being said, given that we already have safeguards provided by Swift here for those kind of mistakes, I think it's more important to make the code readable (`if foo == 42` reads better imho) rather than… what? not ever more safe, as it's already protected against `if foo = 42` mistakes, so…

Reply:
Uhm, `yoda_condition` enforces that the variable comes first, the constant later, which is what you want?

Reply:
Oh. Right. I misread the rule description, especially given that in C-based languages, what we usually call writing "Yoda Condition" is writing `1 == foo` (yoda style) instead of `foo == 1` to avoid the risk of `=` instead of `==`… So I thought given the rule name that it expected us to write in yoda style…

But in fact you're right (I guess the rule is misleading), so let's keep it 👍